lib/es/management/commands/reindex.py

```diff
@@ -31,22 +31,34 @@
 
 # The subset of settings.ES_INDEXES we are concerned with.
 ES_INDEXES = settings.ES_INDEXES
+
+# We have this mapping so we can pass which indexer to use to celery.
+INDEXER_MAP = {
+    'webapp': WebappIndexer,
```

Comment:
Yeah, something like that. Maybe even get rid of `INDEXES` and just set the chunk size in each indexer? Almost all of them default to 500 anyway, so you could set a default value in the base indexer and just override it when necessary